ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/436__test-run-data-science-pipelines-operator-e2e-tests.robot

```diff
@@ -0,0 +1,52 @@
+*** Settings ***
+Documentation     Data Science Pipelines Operator E2E tests - https://github.com/opendatahub-io/data-science-pipelines-operator/tree/main/tests
+Suite Setup       Prepare Data Science Pipelines Operator E2E Test Suite
+Suite Teardown    Teardown Data Science Pipelines Operator E2E Test Suite
+Library           OperatingSystem
+Library           Process
+Resource          ../../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
+Resource          ../../../Resources/RHOSi.resource
+Library             ../../../../libs/DataSciencePipelinesAPI.py
+
+
+*** Variables ***
+# For the initial commit we are hardcoding those environment variables
+${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_DIR}                /tmp/data-science-pipelines-operator
+${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_REPO_URL}           https://github.com/opendatahub-io/data-science-pipelines-operator.git
+${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_REPO_BRANCH}        c9e336df3e6cc8b8e03af2e951dfa54143790339
+${DSPANAMESPACE}                                          dspa-e2e
+${KUBECONFIGPATH}                                         %{HOME}/.kube/config
+
+#robocop: disable: line-too-long
+*** Test Cases ***
+Run Data Science Pipelines Operator E2E Test
+    [Documentation]    Run Data Science Pipelines Operator E2E Test
+    [Tags]
+    ...     Sanity
+    ...     DataSciencePipelines
+    ...     Tier1
+    ${openshift_api}    Get Openshift Server
+    Log    ${openshift_api}
+    ${return_code}    ${output}    Run And Return Rc And Output    cd ${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_DIR} && make integrationtest K8SAPISERVERHOST=${openshift_api} DSPANAMESPACE=${DSPANAMESPACE} KUBECONFIGPATH=${KUBECONFIGPATH}
+    Log    ${output}
+    Should Be Equal As Integers	${return_code}	 0  msg= Run Data Science Pipelines Operator E2E Test failed
```

Comment:
I think you're missing a space (or three) here between the keyword name and the first argument

Reply:
Done